src/ui/server/helpers/pagination/get-total-pages/index.test.ts

```diff
@@ -7,7 +7,7 @@ describe('server/helpers/pagination/get-total-pages', () => {
       const mockTotalApplications = MAX_APPLICATIONS_PER_PAGE + 1;
       const result = getTotalPages(mockTotalApplications);
 
-      const expected = Math.round(mockTotalApplications / MAX_APPLICATIONS_PER_PAGE);
+      const expected = Math.ceil(mockTotalApplications / MAX_APPLICATIONS_PER_PAGE);
```

Comment:
Could there be risk of getting a higher number i.e. 
```javascript
Math.ceil(8.000000000000001) => 9
Math.round(8.000000000000001) => 8
```

Is above a possibility?

Reply:
@abhi-markan I if understand the concern correctly - no, the `totalApplications` will always be a whole number, it's essentially a `.length` from the API. `MAX_APPLICATIONS_PER_PAGE` is a constant set at 15. So this will never having anything other than whole numbers.

We have a lot of tests for making sure the total pages/pagination items are rendered correctly with the following:

- 15 applications (no pagination)
- 16 applications (this is when pagination starts displaying)
- 45 applications
- 60 applications
- 75 applications
- 90 applications
- 135 applications

Reply:
Understood thanks for the clarification.